activation/atxdb.go

```diff
@@ -422,8 +422,9 @@ func (db *DB) SyntacticallyValidateAtx(atx *types.ActivationTx) error {
 		}
 	} else {
 		publicationEpoch := atx.PubLayerID.GetEpoch()
-		if !publicationEpoch.IsGenesis() {
-			return fmt.Errorf("no positioning atx found")
+		fmt.Printf("epoch: %v\n", publicationEpoch)
```

Comment:
I assume this was for local debugging and should be removed. Otherwise, replace it with a proper log print.

Reply:
Yes, this was for local debugging and I forgot to remove it